policy.go

```diff
@@ -55,6 +55,9 @@ type Policy interface {
 	// GetConditions returns the policies conditions.
 	GetConditions() Conditions
 
+	// GetMeta returns the policies metadata as serialized json.
```

Comment:
Maybe add here that metadata can be arbitrary data set by the user? Also I think it would be ok to return either `interface{}` here or `map[string]interface{}` or optionally an `io.Reader` that can be used for `json.Encode/Decode`

Reply:
What do you think of the comment "GetMeta returns the policies arbitrary metadata set by the user."?